src/components/card/examples/card-image/index.njk

```diff
@@ -41,9 +37,7 @@
                 "url": '#0',
                 "text": 'How we keep your data safe and what happens to your personal information.',
                 "image": {
-                    "smallSrc": '/img/small/placeholder-card.png',
-                    "largeSrc": '/img/large/placeholder-card.png',
-                    "alt": 'Image placeholder'
+                    "placeholderURL": ''
```

Comment:
@jrbarnes9 do we need to pass in an empty string here?

Reply:
I've reverted the macro changes I made to allow for a placeholder without setting an image, and raised a new issue (#2238) to improve this separately